R/whatWQPdata.R

```diff
@@ -31,7 +31,7 @@ whatWQPsamples <- function(...){
   baseURL <- appendDrURL(baseURL, mimeType = "tsv")
   
   withCallingHandlers({
-    retval <- importWQP(baseURL, zip=values["zip"] == "yes")
+    retval <- importWQP(baseURL, zip=values["zip"] == "yes", convertType=convertType)
```

Comment:
I don't know where this convertType value would come from. I think we want to add it to the function itself (?). So something like this:
```r
whatWQPsamples <- function(..., convertType = TRUE){

```
Then you'd need to add the @param stuff to this function.